index.d.ts

```diff
@@ -0,0 +1,171 @@
+import { PluginFunction, VueConstructor, Component } from 'vue';
+
+interface CommonOptions {
+  /**
+   *  Position of the toast on the screen.
+   * 
+   *  Can be any of top-right, top-center, top-left, bottom-right, bottom-center, bottom-left.
+   */ 
+  position?: 'top-right' | 'top-center' | 'top-left' | 'bottom-right' | 'bottom-center' | 'bottom-left';
+  /**
+   *  Position of the toast on the screen.
+   * 
+   *  Can be any of top-right, top-center, top-left, bottom-right, bottom-center, bottom-left.
+   */ 
+  draggable?: boolean;
+  /**
+   *  By how much of the toast width in percent (0 to 1) it must be dragged before being dismissed.
+   */
+  draggablePercent?: boolean;
```

Comment:
`draggablePercent` is actually a float that goes from 0 to 1, so `number` is probably the right type
```suggestion
  draggablePercent?: number;
```